sdk/textanalytics/ai-text-analytics/test/README.md

```diff
@@ -15,6 +15,36 @@ To run the live tests, you will also need to set the below environment variables
 - `TEXT_ANALYTICS_API_KEY_ALT` (optional): The secondary API key of the Text Analytics API in your Azure Cognitive Services account.
 - `ENDPOINT`: The endpoint of your Text Analytics API in your Azure Cognitive Services account.
 
-The live tests in this project will create collections in the provided Azure Cognitive Services account.
+In addition to the environment variables above, an Azure Active Directory identity configuration is required. See the [README file for the @azure/identity package](https://github.com/Azure/azure-sdk-for-js/tree/master/sdk/identity/identity) for more details on how to configure an identity for local testing. The test configuration uses the `DefaultAzureCredential` to request an authentication token, so any authentication method that is exposed by `DefaultAzureCredential` will be sufficient to run the tests. The next section will explain how to grant access to your Cognitive Services account to the user or application you wish to use for testing.
```

Comment:
Hi! Yes, we should move this to something like `/documentation/AAD-Authentication.md`. @ramya-rao-a Hi! Thoughts? Then I could move all of these references to that file.

Reply:
@sadasant Are you referring to the text above this comment or below?

Since there are role related information that can change from service to service, the parts below this comment should live in each test read me instead of a central one.

The text above this comment does not add much value in my opinion. One need not know any of it to run the tests. All that is needed is setting up the right env vars which is covered below